apps/s_server.c

```diff
@@ -1506,7 +1506,7 @@ int s_server_main(int argc, char *argv[])
             session_id_prefix = opt_arg();
             break;
         case OPT_ENGINE:
-            engine = setup_engine(opt_arg(), 1);
+            engine = setup_engine_flags(opt_arg(), s_debug, 0);
```

Comment:
This looks like the wrong argument order

Reply:
Oops, good spotting, fixed.